src/tsickle.ts

```diff
@@ -678,32 +678,17 @@ class Annotator extends ClosureRewriter {
       case ts.SyntaxKind.MethodDeclaration:
       case ts.SyntaxKind.GetAccessor:
       case ts.SyntaxKind.SetAccessor:
-        let fnDecl = <ts.FunctionLikeDeclaration>node;
-        let tags = hasExportingDecorator(node, this.typeChecker) ? [{tagName: 'export'}] : [];
+        const fnDecl = node as ts.FunctionLikeDeclaration;
+        const tags = hasExportingDecorator(node, this.typeChecker) ? [{tagName: 'export'}] : [];
 
         if (!fnDecl.body) {
           if (hasModifierFlag(fnDecl, ts.ModifierFlags.Abstract)) {
-            this.emitFunctionType([fnDecl], tags);
-            // Abstract functions look like
-            //   abstract foo();
-            // Emit the function as normal, except:
-            // - remove the "abstract"
-            // - change the return type to "void"
-            // - replace the trailing semicolon with an empty block {}
-            // To do so, skip all modifiers before the function name, and
-            // emit up to the end of the parameter list / return type.
-            if (!fnDecl.name) {
-              // Can you even have an unnamed abstract function?
-              this.error(fnDecl, 'anonymous abstract function');
-              return false;
-            }
-            this.writeRange(fnDecl.name.getStart(), fnDecl.parameters.end);
-            this.emit(') {}');
-            return true;
+            // Emit normally. Declaring the type is handled in visitClassDeclaration below.
+            return false;
           }
```

Comment:
It appears you can delete this 'if' statement because it always returns false after anyway.